client/client.go

```diff
@@ -801,26 +785,37 @@ func (r *NotaryRepository) Update(forWrite bool) error {
 // Returns a tufclient.Client for the remote server, which may not be actually
 // operational (if the URL is invalid but a root.json is cached).
 func (r *NotaryRepository) bootstrapClient(checkInitialized bool) (*tufclient.Client, error) {
-	var (
-		rootJSON   []byte
-		err        error
-		signedRoot *data.SignedRoot
-	)
+	minVersion := 0
```

Comment:
[@riyaz's previous comment](https://github.com/docker/notary/pull/580#discussion_r60459534) got collapsed, so repeating it here:

> Also cc: @endophage - unless I'm missing something, it seems that we can be stricter about version checks and enforce that all versions are >= 1 right?